src/Sylius/Behat/Page/Shop/Order/ShowPage.php

```diff
@@ -52,7 +52,7 @@ public function getRouteName(): string
         return 'sylius_shop_order_show';
     }
 
-    public function getNumberOfItems(): int
+    public function getAmountOfItems(): int
```

Comment:
It looks like `amount` is a proper word here: https://www.quickanddirtytips.com/education/grammar/amount-versus-number